azext_edge/edge/providers/rpsaas/adr2/command.md

```diff
@@ -0,0 +1,476 @@
+swagger:
+https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/deviceregistry/resource-manager/Microsoft.DeviceRegistry/preview/2024-07-01-preview/deviceregistry.json
+
+## Assets
+id: `/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.DeviceRegistry/assets/my-asset`
+
+discovered assets are a different thing but have the same props: `/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.DeviceRegistry/discoveredAssets/my-asset`
```

Comment:
what are your thoughts on combining assets vs discovered assets?

If we do so, this will require some mixing, like in the show here:
![image](https://github.com/user-attachments/assets/1c6aa958-7656-463e-bf25-d6510bc2de25)

The discovered will be used to determine which operations class to use